tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-	py{35,36,37,38,39}-django{111,20,21,22}-apns{030,060}
-	py{36,37,38,39}-django30-apns{030,060}
+	py{35,36,37,38,39}-django{22}-apns{030,060}
```

Comment:
Jazzbands general approach has been to drop support for Python 3.5.

Reply:
That is not a problem. Just updated without python 3.5.